scripts/change_url

```diff
@@ -114,13 +114,21 @@ fi
 ynh_add_config --template="homeserver.yaml" --destination="/etc/matrix-$app/homeserver.yaml"
 ynh_add_config --template="log.yaml" --destination="/etc/matrix-$app/log.yaml"
 
+#=================================================
+# SECURE FILES AND DIRECTORIES
+#=================================================
+
+# Only setting permissions for the two config files updated above
+chown $synapse_user:root -R /etc/matrix-$app
+chmod u=rwX,g=rX,o= -R /etc/matrix-$app
+
 #=================================================
 # SETUP PERMISSIONS
 #=================================================
 
 ynh_script_progression --message="Configuring permissions..." --weight=1
 ynh_permission_url --permission=server_api --clear_urls
-ynh_permission_url --permission=server_api --url=$domain/_matrix --additional_urls=$server_name/.well-known/matrix \
+ynh_permission_url --permission=server_api --url=$domain/_matrix --add_url=$server_name/.well-known/matrix \
```

Comment:
I'm not sure to understand your comment. I was merely fixing the call to the helper since `--additional_urls` is not a valid flag for it: https://yunohost.org/en/packaging_apps_helpers#ynh-permission-url